R/aes-calculated.r

```diff
@@ -54,6 +96,40 @@ is_calculated <- function(x) {
     stop("Unknown input:", class(x)[1])
   }
 }
+is_mapped <- function(x) {
+  if (is.atomic(x)) {
```

Comment:
I would merge all the `FALSE` branches into one here. I don't think the `is.pairlist()` branch is needed.